lib/api/client-commands/registerBasicAuth.js

```diff
@@ -0,0 +1,50 @@
+const ClientCommand = require('./_base-command.js');
+const {Logger} = require('../../utils');
+
+/**
+ * Automate the input of basic auth credentials whenever they arise.
+ *
+ * @example
+ *  this.demoTest = function (browser) {
+ *    browser.registerBasicAuth(1000, 800);
+ *    browser.url('https://page-needs-auth-credentials.com');
+ *  };
+ *
+ *
+ * @method registerBasicAuth
+ * @param {string} username
+ * @param {string} password
+ * @param {function} [callback] Optional callback function to be called when the command finishes.
+ */
+class RegisterBasicAuth extends ClientCommand {
+
+  performAction(callback) {
+
+    const {browserName}  = this.api.capabilities;
+    if (browserName!=='chrome' && browserName!=='edge') {
```

Comment:
We can use `this.api.isChrome() / isEdge()` here.

Reply:
done